Makefile.sp

```diff
@@ -114,7 +115,24 @@ sp-vale: sp-install
 	@echo ""
 	@. $(VENV); vale --config "$(SPHINXDIR)/vale.ini" --glob='*.{md,txt,rst}' $(TARGET)
 
-
+sp-pdf-prep: sp-install
+	@for packageName in $(REQPDFPACKS); do (dpkg-query -W -f='$${Status}' $$packageName 2>/dev/null | \
+      grep -c "ok installed" >/dev/null && echo "Package $$packageName is installed") || \
+      (echo "PDF generation requires the installation of the following packages: $(REQPDFPACKS)" && \
+      echo "" && echo "sudo make pdf-prep-force will install these packages" && echo "" && echo \
+      "Please be aware these packages will be installed to your system" && false) ; done
```

Comment:
Is the different indentation intended/needed?

Reply:
I added it for readability - so it was clear it's the one command. It's not needed, and can be removed.

Reply:
```suggestion
        grep -c "ok installed" >/dev/null && echo "Package $$packageName is installed") || \
        (echo "PDF generation requires the installation of the following packages: $(REQPDFPACKS)" && \
        echo "" && echo "sudo make pdf-prep-force will install these packages" && echo "" && echo \
        "Please be aware these packages will be installed to your system" && false) ; done
```